src/users/dto/create-user.dto.ts

```diff
@@ -0,0 +1,30 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { IsEmail, IsEnum, IsNotEmpty, IsString, MinLength} from "class-validator";
+import { UserRole } from "src/constants";
+import { isUniqueDb } from "@youba/nestjs-dbvalidator";
+import { Transform } from "class-transformer";
+
+export class CreateUserDto {
+    @ApiProperty({ example: 'John', required: true })
+    @IsNotEmpty()
+    @IsString()
+    name: string;
+
+    @ApiProperty({ example: 'ronaldo@gmail.com', required: true})
+    @IsNotEmpty()
+    @isUniqueDb({table : 'user', column: 'email', message: 'Email already exists'})
```

Comment:
Debes instalar un prettier
```suggestion
    @isUniqueDb({ table: 'user', column: 'email', message: 'Email already exists' })
```

Reply:
Corregido papax